### ISSUE TITLE
Questions about "vertices_z" optimation 

### ISSUE THREAD

**Issue by luohongcheng** [DRosemei/RoMe#24]
Hello, in https://github.com/DRosemei/RoMe/blob/main/models/voxel.py#L313, the function ".detach()" is used, does it mean the value of vertices_z and the height MLP network would not be updated?

**Comment by DRosemei** [comment]
Hi, yes it is. Only update a small area according to the waypoint.

**Comment by nkliuhui** [comment]
请问怎么打开梯度传播，对MLP进行优化呢？对torch不太了解，改完无法确定对不对
